### PR TITLE
More async Liquid rendering docs

### PR DESCRIPTION
**Pull request by antonpaisov**
This was requested by @iurevych

### CODE REVIEW

File: source/campaigns/editor/filters.rst [hunk 1]
```diff
@@ -746,3 +746,27 @@ Rendered Liquid
...
+
+  Talkable.loadLiquid("async_advocate_info", function(rendered_template) {
+    console.log(rendered_template);
+  });
+
+Rendered Liquid
+
+.. code-block:: text
+
+  {"first_name":null,"last_name":null,"email":"advocate@example.com","username":null,"external_customer_id":null,"opted_in_at":null,"sub_choice":false,"purchases_sum":0.0,"purchases_count":0,"events_count":0,"events_count_by_category":{"friend_signup":0,"purchase":0},"referrals_count":0,"referred_by":null,"custom_properties":{},"unsubscribed":false}
```
Comment (iurevych) [inline source/campaigns/editor/filters.rst]: Can we tweak code formatting so it reads easier?

File: source/campaigns/editor/filters.rst [hunk 2]
```diff
@@ -746,3 +746,27 @@ Rendered Liquid
 .. code-block:: text
 
   "Whatever Liquid you would like to have"
+
+Example 4
+~~~~~~~~~
+If you want to unwrap nested lazy variables:
```
Comment (iurevych) [inline source/campaigns/editor/filters.rst]: Not sure I get it, how about:
``` rst
Use Liquid `json` filter to transform deeply nested object structure into a valid JSON.
```

File: source/campaigns/editor/filters.rst [hunk 3]
```diff
@@ -746,3 +746,27 @@ Rendered Liquid
...
+
+.. code-block:: javascript
+
+  Talkable.loadLiquid("async_advocate_info", function(rendered_template) {
+    console.log(rendered_template);
+  });
+
+Rendered Liquid
+
+.. code-block:: text
```
Comment (zhuravel) [inline source/campaigns/editor/filters.rst]:
``` suggestion
.. code-block:: javascript

File: source/campaigns/editor/filters.rst [hunk 4]
```diff
@@ -746,3 +746,27 @@ Rendered Liquid
 .. code-block:: text
 
   "Whatever Liquid you would like to have"
+
+Example 4
+~~~~~~~~~
```
Comment (zhuravel) [inline source/campaigns/editor/filters.rst]: Please keep a blank line below the header.

File: source/campaigns/editor/filters.rst [hunk 5]
```diff
@@ -746,3 +746,47 @@ Rendered Liquid
...
+
+Use ``json`` Liquid filter to transform deeply nested object structure into a valid JSON.
+
+Liquid
+
+.. code-block:: liquid
+
+  {{ 'async_advocate_info' | async_rendering: '[[ advocate_info | json ]]'}}
```
Comment (zhuravel) [inline source/campaigns/editor/filters.rst]:
``` suggestion
  {{ 'async_advocate_info' | async_rendering: '[[ advocate_info | json ]]' }}

**Pull request merged**